src/ngraph/runtime/cpu/pass/cpu_fusion.cpp

```diff
@@ -1403,3 +1401,211 @@ void ngraph::runtime::cpu::pass::CPUFusion::construct_bounded_relu()
     auto m = std::make_shared<pattern::Matcher>(min, callback);
     this->add_matcher(m);
 }
+
+void ngraph::runtime::cpu::pass::CPUFusion::construct_conv_bias_folded_batch_norm()
+{
+    Shape shape{2, 2, 1, 1};
+    auto input = std::make_shared<pattern::op::Label>(element::f32, Shape{2, 2, 1, 1});
```

Comment:
It looks like input, filters were meant to use `shape`

Reply:
Removed shape.